Makefile

```
@@ -1,10 +1,14 @@
 PKG := github.com/btcsuite/btcwallet
+TOOLS_DIR := tools
+
+GOCC ?= go
 
 LINT_PKG := github.com/golangci/golangci-lint/cmd/golangci-lint
 GOACC_PKG := github.com/ory/go-acc
-GOIMPORTS_PKG := golang.org/x/tools/cmd/goimports
+GOIMPORTS_PKG := github.com/rinchsan/gosimports/cmd/gosimports
 
 GO_BIN := ${GOPATH}/bin
```

```suggestion
GO_BIN := $$(go env GOBIN)
```

please, use the `go env GOBIN` command to dicover the go bin setup, in my case the path is inside another package manager called mise.

Works well here with this change.

---

> please, use the `go env GOBIN` command to dicover the go bin setup, in my case the path is inside another package manager called mise.
> 
> Works well here with this change.

Addressed

---

https://github.com/btcsuite/btcd/pull/2320#discussion_r2287153465

---

> [btcsuite/btcd#2320 (comment)](https://github.com/btcsuite/btcd/pull/2320#discussion_r2287153465)

Addressed